packages/core/src/icon/icon.stories.mdx

```diff
@@ -24,6 +24,17 @@ ClarityIcons.addIcons(userIcon);
 <cds-icon shape="user"></cds-icon>
 ```
 
+TypeScript users may use the following in contexts where the markup is being type-checked (e.g. in TSX):
```

Comment:
This wording is a little arbitrary, please feel free to ask me for corrections!

Reply:
Actually in can be used in JSX as well.

I would suggest something similar to 

```
To use the default by Clarity icons shapes import the icon name from `@clr/core/icon`
```

Reply:
Thank you very much, @bbogdanov and @mathisscott, done!

@bbogdanov, I wasn't entirely following the meaning of your sentence, so I worded it a little differently while taking your feedback into consideration. Does this work for you, or do you want me to edit further?